### PR TITLE
Fixed reference for fee docs, ethers interaction

### PR DESCRIPTION
**Pull request by Nicca42**
Reference fee code did not call ethers correctly.

### CODE REVIEW

**Review by elenadimitrova — APPROVED** [review]

File: src/docs/developers/l2/new-fees.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ This transaction is typical. In almost all cases **L2 execution fee** is negligi
       ```jsx
       import { getContractFactory, predeploys }from '@eth-optimism/contracts'
       import { ethers } from 'ethers'
-      const OVM_GasPriceOracle = getContractFactory('OVM_GasPriceOracle')
+      const OVM_GasPriceOracle = ethers.getContractFactory('OVM_GasPriceOracle')
```
Comment (K-Ho) [inline src/docs/developers/l2/new-fees.md:47]: we're using `getContractFactory` from `@eth-optimism/contracts`

File: src/docs/developers/l2/new-fees.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ This transaction is typical. In almost all cases **L2 execution fee** is negligi
...
             .attach(predeploys.OVM_GasPriceOracle)
       const WETH = new Contract(...) //Contract with no signer
       const unsignedTx = WETH.populateTransaction.transfer(to, amount)
-      const serializedTx = serialize({
+      const serializedTx = ethers.utils.serializeTransaction({
```
Comment (K-Ho) [inline src/docs/developers/l2/new-fees.md:51]: https://github.com/ethereum-optimism/optimism/blob/b7e9d10c77b554f6635aa0ef698be46e8991ffde/integration-tests/test/fee-payment.spec.ts#L7 - I think it's actually  
```
import { serialize } from '@ethersproject/transactions'
```

**Review by K-Ho — CHANGES_REQUESTED** [review]
Needs some changes

File: src/docs/developers/l2/new-fees.md [hunk 3]
```diff
@@ -43,6 +43,7 @@ This transaction is typical. In almost all cases **L2 execution fee** is negligi
 
       ```jsx
       import { getContractFactory, predeploys }from '@eth-optimism/contracts'
+      import { serialize } from '@ethersproject/transactions'
       import { ethers } from 'ethers'
```
Comment (K-Ho) [inline src/docs/developers/l2/new-fees.md:47]:
```suggestion
      import { Contract } from 'ethers'
```

**Review by K-Ho — CHANGES_REQUESTED** [review]
one more fix

**Review by K-Ho — APPROVED** [review]

**Pull request merged**